src/__screenshot_tests__/form-select-screenshot-test.tsx

```diff
@@ -20,7 +20,7 @@ test('Select - appears properly with selected option on DESKTOP', async () => {
         device: 'DESKTOP',
     });
 
-    const field = await screen.findByLabelText('Select a fruit');
+    const field = await screen.findByRole('button');
```

Comment:
I don't know why, but if we try to do `await field.click()` without this change, the test fails because of the element being detached from the DOM:

![image](https://github.com/Telefonica/mistica-web/assets/25785151/20dd7f59-a301-40da-86f9-4e10e3df520b)